3-UsingAPIs/Week2/homework/ex3-rollAnAce.js

```diff
@@ -12,21 +12,23 @@ Full description at: https://github.com/HackYourFuture/Homework/blob/main/3-Usin
 // ! Do not change or remove the next two lines
 const rollDie = require('../../helpers/pokerDiceRoller');
 
-function rollDieUntil(wantedValue) {
-  // TODO: rewrite this function using async/await
-  return rollDie().then((value) => {
-    if (value !== wantedValue) {
-      return rollDieUntil(wantedValue);
+async function rollDieUntil(wantedValue) {
+  const loop = true;
+  while (loop) {
```

Comment:
What if we would change this code to this:

```suggestion
  let value = null;
  while (value !== wantedValue) {
```

How would change the rest of the code to make this work?